tests/integration_tests/security/test_seccomp.py

```diff
@@ -11,11 +11,11 @@
 import host_tools.cargo_build as host  # pylint:disable=import-error
 
 
-@pytest.fixture
-def tmp_basic_jailer(test_session_root_path):
-    """Build `demo_basic_jailer`, required for the basic seccomp tests.
+@pytest.fixture(scope='session')
```

Comment:
nit: rephrase the the commits that change the seccomp integration tests in a similar way. i.e:
* seccomp integ test: ...
* seccomp integ tests: ...
* seccomp tests: ...